src/database/models/index.js

```diff
@@ -0,0 +1,38 @@
+
+import * as config from '../config/config';
+
+const fs = require('fs');
+const path = require('path');
+const Sequelize = require('sequelize');
+
+const basename = path.basename(__filename);
+const env = process.env.NODE_ENV || 'development';
+
+const db = {};
+
+let sequelize;
+
+if (config[env].use_env_variable) {
+  sequelize = new Sequelize(config[env].url);
```

Comment:
use options instead of using url only. Dialect should be explicitly supplied

Reply:
It's funny -- I left this this way because Sequelize scaffolded it to give options with the environment variable. I'll update to remove the logic.